cmd/stork/stork.go

```diff
@@ -66,6 +66,9 @@ const (
 	cmName                     = "stork-version"
 	eventComponentName         = "stork"
 	debugFilePath              = "/var/cores"
+	awsKopiaExecutorImage      = "709825985650.dkr.ecr.us-east-1.amazonaws.com/portworx/kopiaexecutor"
```

Comment:
Along with the default values for these images, lets add the support to read from the config map too.

Reply:
Those are already present as part of this function
```
// KopiaExecutorImage returns a docker image that contains kopiaexecutor binary.
func KopiaExecutorImage() string {
	if customImage := strings.TrimSpace(GetConfigValue(drivers.KopiaExecutorImageKey)); customImage != "" {
		return customImage
	}
	// use a versioned docker image
	return strings.Join([]string{drivers.KopiaExecutorImage, version.Get().GitVersion}, ":")
}
```